mlrun/db/httpdb.py

```diff
@@ -4012,9 +4020,11 @@ def store_alert_config(
     def get_alert_config(self, alert_name: str, project="") -> AlertConfig:
         """
         Retrieve an alert.
+
         :param alert_name: The name of the alert to retrieve.
-        :param project: The project that the alert belongs to.
-        :return: The alert object.
+        :param project:    The project that the alert belongs to.
+
+        :returns:           The alert object.
```

Comment:
Sometime you use `return` and sometimes `returns`. What is the correct format and are both options displayed differently?

Reply:
Updated all to [returns](url)